package.json

```diff
@@ -51,6 +51,7 @@
     "dependencies": {
         "@microsoft/powerquery-formatter": "0.1.5",
         "@microsoft/powerquery-parser": "0.9.4",
+        "vscode-languageserver": "8.0.1",
```

Comment:
We need the full language server package? We can't just get this from the vscode-languageserver-types package?